test/ASTMerge/injected-class-name-decl-1/test.cpp

```diff
@@ -0,0 +1,6 @@
+// Trigger a case when at import of template record and replacing its type
+// with InjectedClassNameType there is a PrevDecl of the record.
+
+// RUN: %clang_cc1 -std=c++1z -emit-pch -o %t.ast %S/Inputs/inject1.cpp
+// RUN: %clang_cc1 -std=c++1z -emit-obj -o /dev/null -ast-merge %t.ast %S/Inputs/inject2.cpp
```

Comment:
Could we remove `Inputs/inject2.cpp` and instead use `test.cpp` as the "to" context:
```
// RUN: %clang_cc1 -std=c++1z -emit-obj -o /dev/null -ast-merge %t.ast %s
```
?

Reply:
Do we need the emit-obj? Can't we just add -fsyntax-only or something like this?